walk-in-interview/src/main/java/com/google/job/data/JobDuration.java

```diff
@@ -0,0 +1,37 @@
+package com.google.job.data;
+
+/** Enumeration for duration of a job. */
+public enum JobDuration {
+    ONE_WEEK("ONE_WEEK"),
```

Comment:
Look like some times we have string presentation of the enum as all-cap, some times as train-case. Is that intended?

Reply:
Yeah intended; for this all-caps, is to match with the id in front end, while other train-case, is for human to read